### PR TITLE
Convert TestDagRunsEndpoint to use module fixture

### PR DESCRIPTION
**Pull request by uranusjr**
Quick gain similar to #14746.

### CODE REVIEW

**Review by kaxil — APPROVED** [review]

File: tests/www/api/experimental/test_dag_runs_endpoint.py [hunk 1]
```diff
@@ -27,10 +28,26 @@
 from tests.test_utils.decorators import dont_initialize_flask_app_submodules
 
 
-class TestDagRunsEndpoint(unittest.TestCase):
-    @classmethod
-    def setUpClass(cls):
-        super().setUpClass()
+@pytest.fixture(scope="module")
```
Comment (sfc-gh-kbregula) [inline tests/www/api/experimental/test_dag_runs_endpoint.py:31]: My fear is that the module never closes, so this creates a side effect between all tests. Am I right?

Comment (uranusjr) [reply tests/www/api/experimental/test_dag_runs_endpoint.py:31]: My understanding is, the fixuture’s lifeline is not tied to the module *itself*, but pytest internally tracks whether all tests in the module and tears down the fixture after they all run. `scope` means more like “tests in this scope” in the lexical sense, not the actual function/class/module the fixture lives in.

Comment (ashb) [reply tests/www/api/experimental/test_dag_runs_endpoint.py:31]: Yeah, this is not tied to the module being imported/unimported, but to what tests Pytest has left to run.

**Review by ashb — APPROVED** [review]

**Comment by ashb** [comment]
Nice. 48s -> 10s 😁

**Pull request merged**